java/arcs/jvm/storage/database/testutil/FakeDatabaseManager.kt

```diff
@@ -39,7 +39,7 @@ import kotlinx.coroutines.sync.Mutex
 import kotlinx.coroutines.sync.withLock
 
 /** [DatabaseManager] which generates fake [Database] objects. */
-class FakeDatabaseManager : DatabaseManager {
+open class FakeDatabaseManager : DatabaseManager {
```

Comment:
you also need to open the function you want to override: getEntitiesCount getStorageSize isStorageTooLarge
by the way, I think you can just do this changes in your cl, does not need to be a separate pr